src/base/base.pri

```diff
@@ -107,10 +107,10 @@ SOURCES += \
     $$PWD/rss/private/rssparser.cpp \
     $$PWD/utils/fs.cpp \
     $$PWD/utils/gzip.cpp \
+    $$PWD/utils/net.cpp \
```

Comment:
You do not need to fix this unless you want. If so, leave the fix in its own commit, please.